Makefile

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/make
+
+# Credit: @Fosol, @asanchezr for creating and maintaining the bcgov/PIMS 
+SHELL := /usr/bin/env bash
+
+ifneq ($(OS),Windows_NT)
+POSIXSHELL := 1
+else
+POSIXSHELL :=
+endif
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' Makefile | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: help
+
+##############################################################################
+# Docker Development
+##############################################################################
+
+restart: ## Restart local docker environment (n=service name)
+	$(info Restart local docker environment)
+	@make stop n=$(n)
+	@make up n=$(n)
+
+refresh: ## Recreates local docker environment (n=service name)
+	$(info Recreates local docker environment)
+	@make stop n=$(n)
+	@make build n=$(n)
+	@make up n=$(n)
+
+up: ## Runs the local containers (n=service name)
+	@echo "$(P) Running client and server..."
+	@docker-compose up -d $(n)
```

Comment:
One thing to note on Linux CLI, Docker Version 1 (V1) and Version (V2) of docker compose command is different. docker-compose was compatible with V1 and now docker compose works for V2. It might give errors.  Hyphen vs space. Something to note. On windows and Mac this may work just fine.